docs/src/main/sphinx/installation/cli.rst

```diff
@@ -59,7 +59,7 @@ Use ``--help`` to see information about specifying the keystore, truststore, and
 other authentication details as required. If using Kerberos, see :doc:`/security/cli`.
 
 External authentication - SSO
------------------------------
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Isnt this going to conflict with the other PR?

Reply:
Maybe. It's the exact same edit, so any good merge-diff algo will recognize that.